media/videostitcher/stitcher_test.go

```diff
@@ -35,14 +35,19 @@ const (
 	slateID             = "my-go-test-slate"
 	deleteSlateResponse = "Deleted slate"
 
-	deleteCdnKeyResponse = "Deleted CDN key"
-	gcdnCdnKeyID         = "my-go-test-google-cdn"
-	akamaiCdnKeyID       = "my-go-test-akamai-cdn"
-	hostname             = "cdn.example.com"
-	updatedHostname      = "updated.example.com"
-	gcdnKeyname          = "gcdn-key"
-	privateKey           = "VGhpcyBpcyBhIHRlc3Qgc3RyaW5nLg=="
-	updatedPrivateKey    = "VGhpcyBpcyBhbiB1cGRhdGVkIHRlc3Qgc3RyaW5nLg=="
+	deleteCdnKeyResponse      = "Deleted CDN key"
+	mediaCdnKeyID             = "my-go-test-media-cdn"
+	cloudCdnKeyID             = "my-go-test-cloud-cdn"
+	akamaiCdnKeyID            = "my-go-test-akamai-cdn"
+	hostname                  = "cdn.example.com"
+	updatedHostname           = "updated.example.com"
+	keyName                   = "my-key"
+	mediaCdnPrivateKey        = "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxzg5MDEyMzQ1Njc4OTAxMjM0NTY3DkwMTIzNA"
```

Comment:
Would it make sense to store these keys in Secret Manager?

Reply:
These are random strings used as fake keys and are immediately deleted.

Reply:
Then it's probably clearer and safer to generate these on the fly during the test -- you can use a UUID or some other random string right?

Reply:
Done - generating UUIDs